hw/ip/pinmux/data/pinmux.hjson.tpl

```diff
@@ -29,7 +29,7 @@
   dv_doc:             "../doc/dv",
   hw_checklist:       "../doc/checklist",
   sw_checklist:       "/sw/device/lib/dif/dif_pinmux",
-  version:            "1.0.0",
+  version:            "1.1.0",
```

Comment:
Increasing minor version because _input disable attribute_ got added for non-manual pads